app/src/main/java/dev/ukanth/ufirewall/MainActivity.java

```diff
@@ -176,6 +177,7 @@ public void setDirty(boolean dirty) {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        MultiUser.setup();
```

Comment:
This should be called after the root check is done ?

Reply:
Fixed in [c75f211](https://github.com/ukanth/afwall/pull/1349/commits/c75f211c905f0799a859cd56eebe890c6a53b236) hopefully, lemme know if I did it wrong.